### ISSUE TITLE
Wtyczka Wizja TV

### ISSUE THREAD

**Issue by wawel3** [mrknow/filmkodi#705]
## Please follow the guide below / Podążaj za przewodnikiem
- Put an "x" into all boxes [ ] relevant to your *issue* (Wstaw "x" w każde pole [ ], które dotyczy twojego zgłoszenia)
- Use *Preview* tab to see how your issue will actually look like (Użyj *Perview* aby sprawdzić, czy wszystko wygląda dobrze)

---
## Kodi & OS
- Kodi verison: _____16.1
- OS version: _____Win 7 and android 4.4

---
## Addon / Wtyczka
- [ ] Filmy online - Mrknow
- [ x] MrknowTV
- [ ] Specto
- [ ] FanFILM
- Version / Wersja: 2016.12.09.2

---
## Issue / Zgłoszenie
- [ x] Bug report / Błąd
- [ ] Site request / Prośba o nową stronę
- [ x] Feature request / Prośba o nową funkcję
- [ ] Question / Pytanie
- [ ] Other / Inne

---
## Descryption / Opis
We wtyczce MrknowTV przestala dzialac WizjaTV, czy ktos wie jak to mozna naprawic?

**Comment by wawel3** [comment]
Siadla WizjaTV we wtyczce MrknowTV, prawdopodobnie przez zmiane na stronie wizjatv, dopiero po nacisnieciu captcha otwiera sie glowna strona. Czy ktos wie jak to naprawic.

**Comment by wacek-11** [comment]
witam,
nie przestala dzialac tylko "captcha" blokuje wizje.tv...przeczytaj moj wpis o "captcha"...ci co robia mrknowtv musza zmienic ustawienia aby nie blokowalo to w kodi. Jak narazie nikt nie zareagowal na moj wpis

**Comment by mrknow** [comment]
U mnie działa teraz. Na razie zamykam, ponieważ nie udało mi się powtórzyć błędu

**Comment by wawel3** [comment]
Tu chodzi o wtyczke mrknowTV nie mrknow w ktorej nie ma wizjaTV, w tej pierwszej tak jak wacek-11 pisal to captcha blokuje dostep, mam wykupione premium i juz od kilku tygodni nie moge ogladac w kodi.

**Comment by kamilinho1** [comment]
Zgadza się. Strona internetowa wizji wymaga potwierdzenia captchy. Urządzenia typu Raspberry Pi nie mają przeglądarki internetowej więc nie mają możliwości wyłączenia weryfikacji. Korzystam na RPi z różnych wtyczek na systemie OSMC i po wprowadzeniu captchy żadna wtyczka dla wizji nie działa. Na Windows, czy na androidzie można najpierw wyłączyć captchę przez przeglądarkę internetową. Potem na Kodi wtyczki do wizji normalnie działają. Dlatego problem się nie powtarza. Podsumowując wtyczka nie działa, captcha blokuje wyświetlanie streamów.

**Comment by mrknow** [comment]
captcha ? Dla kont premium też ? 
Napisz jak mogę ją wywołać u siebie ?. To nie problem aby dorobić captche w kodi, tylko muszę wiedzieć jak ją wywołać

**Comment by kamilinho1** [comment]
Tak, mam premium. Korzystam z wizji przede wszystkim na RPi z OSMC. I tam jest ten problem. Po prostu z dnia na dzień wszystkie wtyczki (mtv guide, etv guide, Twoja i kilka innych) przestały pokazywać dostępne streamy. Zorientowałem się że to chodzi o captchę dopiero jak przez smartfona chciałem wejść na stronę wizja.tv i pojawiła się ta weryfikacja. Spróbuj może po prostu wejść na ich stronę przez jakieś inne urządzenie. Przed chwilą próbowałem z innego smartfona i też się to pojawiło.

19.01.2017 napisał:
>captcha ? Dla kont premium też ? 
>Napisz jak mogę ją wywołać u siebie ?. To nie problem aby dorobić
>captche w kodi, tylko muszę wiedzieć jak ją wywołać

**Comment by kamilinho1** [comment]
Screen

19.01.2017 napisał:
>captcha ? Dla kont premium też ? 
>Napisz jak mogę ją wywołać u siebie ?. To nie problem aby dorobić
>captche w kodi, tylko muszę wiedzieć jak ją wywołać

**Comment by mrknow** [comment]
@kamilinho1 - wrzuć link do screena. Tutaj nic nie widzę

**Comment by kamilinho1** [comment]
http://imgur.com/q0EfK0l

21.01.2017 napisał:
>@kamilinho1 - wrzuć link do screena. Tutaj nic nie widzę

**Comment by kamilinho1** [comment]
Dasz radę naprawić? Od tygodnia brak info.

**Comment by wawel3** [comment]
Ja tez mam premium z wizjatv, na stronie internetowej po wylaczeniu captcha i zalogowaniu sie dziala OK, ale w kodi przy wejsciu na Mrknowtv i nacisnieciu wizjatv nawet kanaly sie nie pokazuja. (mam wpisany login name i password pod accounts). To wszystko zaczelo sie po wrowadzeniu przez wizjatv captcha okolo Stycznia 02 2017.

**Comment by maryanek** [comment]
Pisałem do ich supportu, i napisali, ze 'mrKnow musi zmienić sposób działania wtyczki, tak aby używała naszego api' i, ze wysłali mu info dawno temu...

Da się coś z tym zrobić? To trochę uciążliwe :/

**Comment by kamilinho1** [comment]
Nie da się naprawić? Autor olał temat?

**Comment by wawel3** [comment]
Tyle czasu i ne mamy yadnej informacji, czy to sie da naprawic czy nie, szkoda......................

**Comment by wawel3** [comment]
No i jak z tą WizjaTV, czy da się coś zrobić, żeby chodziła w kodi

**Comment by wojtas29** [comment]
U mnie Wizja działa, tylko, kiedy już włączę jakiś kanał i próbuję cofnąć do listy kanałów używając klawisza esc. klodi zamyka się
Poza tym czy jest jakieś epg do wizjaTV?

**Comment by waldi22** [comment]
Działając dodatek PVR Simple Client z listą m3u epg  wizji-tv na wtyczce MrKnowTV premium , bardzo by się przydał i ułatwił życie . Poza tym wszystko śmiga perfekt .

**Comment by wawel3** [comment]
Mam zinstalowany kodi 17.1 na PC a w androidowych box tv  jeszcze 16.1 i w zadnych nawet lista kanalow sie nie pokazuje. Probowalem w MrKnowTV i w osobnej wtyczce do WizjiTV i dalej nic. Niech ktos napisze czy cos trzeba dodatkowego ustawic. W tej chwili nie mam premium z WizjaTV, bo od czasu kiedy wprowadzili captcha na ich stronie internetowej to nie chodzi mi w kodi, ale jestem zarejestrowany na stronie Wizja, bo w MrKnow trzeba wpisac username i password, ale przynajmniej lista kanalow powinna sie pokazac. Jestem ciekaw czy nie dlatego, ze mieszkam w ameryce polnocnej.

**Comment by kamilinho1** [comment]
Napisz do wizji, żeby odblokowali ci IP. Albo zmień zewnętrzny adres. U mnie pomogło a miałem taki sam problem jak ty.

04.05.2017 napisał:
>Mam zinstalowany kodi 17.1 na PC a w androidowych box tv  jeszcze 16.1
>i w zadnych nawet lista kanalow sie nie pokazuje. Probowalem w MrKnowTV
>i w osobnej wtyczce do WizjiTV i dalej nic. Niech ktos napisze czy cos
>trzeba dodatkowego ustawic. W tej chwili nie mam premium z WizjaTV, bo
>od czasu kiedy wprowadzili captcha na ich stronie internetowej to nie
>chodzi mi w kodi, ale jestem zarejestrowany na stronie Wizja, bo w
>MrKnow trzeba wpisac username i password, ale przynajmniej lista
>kanalow powinna sie pokazac. Jestem ciekaw czy nie dlatego, ze mieszkam
>w ameryce polnocnej.

**Comment by Huero23** [comment]
WizjaTv zniknęła? Mr. KNOW ?

**Comment by frijerz** [comment]
Wtyczka dziala na windows10, ale na Androidzie pokazują się kanaly, ale nie odtwarza.

**Comment by IzabellaC** [comment]
Mam ten sam problem. Kanaly Wizja.TV przstaly sie dzis  pokazywac na KODI. Probowalam w MrKnowTV i w osobnej wtyczce do WizjiTVale bez skutku. 
Czy jest to kwestia nowych ustawien?

**Comment by kobra7950gt** [comment]
Wizja,tv nie działa kodi 17.1 POMOCY!!!!

**Comment by Adamek125** [comment]
U nie to samo, na komputerze premium Wizja działa jak trzeba, na android boxie już w ogóle, do tego prawie cały filmy online mrknow nie działa w dziale telewizja, ani jednego kanału nie mogę odtworzyć.
